ext/afform/core/ang/af/afForm.component.js

```diff
@@ -116,7 +116,9 @@
           if (url.indexOf('civicrm/') === 0) {
             url = CRM.url(url);
           } else if (url.indexOf('/') === 0) {
-            url = $location.protocol() + '://' + $location.host() + url;
+            let port = $location.port();
```

Comment:
I see you're itching to use ES6! Again I'm not opposed to it, but I think when we introduce it we ought to do so intentionally (maybe with a dev announcement) rather than just slipping it into an unrelated patch.

Reply:
Oh look at that, we do support ES6 already. Cool I forgot we merged that.